lib/plugins/plugin-validator.js

```diff
@@ -58,13 +59,23 @@ function validateConfigPaths (pluginConfig, executionPath) {
 
     criteriaConfig.forEach((configEntry, index) => {
       try {
-        if (typeof configEntry === 'string' && configEntry[0] === '/') {
-          checkPathExists(executionPath, configEntry, key)
+        if (key === 'pluginDependencies') {
+          if (typeof configEntry.packageName !== 'string') {
+            errors.push(`In section ${key}, the packageName '${configEntry.packageName}' should be a valid package name`)
+          }
+          if (typeof configEntry.minVersion !== 'string') {
```

Comment:
This code suggests that minVersion has to be set, I was expecting both min and max to be optional and to accept strings as well as objects.  e.g. I would see each of these as valid configs

```
"pluginDependencies": ["hmrc-frontend"]
```

```
"pluginDependencies": [{
  "packageName": "@x-govuk/edit-prototype-in-browser",
  "maxVersion": "1.0.0"
}]
```

```
"pluginDependencies": [{
  "packageName": "@x-govuk/edit-prototype-in-browser",
  "minVersion": "1.0.0"
}]
```

```
"pluginDependencies": [{
  "packageName": "@x-govuk/edit-prototype-in-browser",
  "minVersion": "0.1.1",
  "maxVersion": "1.0.0"
}]
```

Reply:
Fixed as requested including some extra unit tests.